modules/packages/ArgumentParser.chpl

```diff
@@ -20,6 +20,22 @@
 /*
   A module to be used for defining and parsing command line arguments.
 
+  .. note::
+
+    You must declare your main method to take an array of string arguments in
+    order to get values from the command line.
+    See the :ref:`quickstart example<argumentParser-quickstart>` below or the
+    technote about :ref:`getting arguments from main()<technote-mainWithArgs>`.
+
+
+  .. note::
+
+    Chapel programs can make use of :ref:`configuration constants and variables <ug-configs>`.
+    The ArgumentParser module can be used in place of, or in addition to,
+    configuration variables and constants. However, many programs may not need
+    the added complexity of the ArgumentParser, relying solely on the
+    functionality provided by configuration variables.
```

Comment:
Perhaps this change?

However, many programs may not need the added complexity of the ArgumentParser, relying solely on the functionality provided by configuration variables.

->

In particular, it is common for Chapel programs to rely solely upon configuration variables and for such programs the ArgumentParser is not needed.